modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -231,12 +231,19 @@
                     {{ $t('apostrophe:automaticTranslationSettings') }}
                   </span>
                 </p>
+                <p
+                  v-if="automaticTranslationDisclaimer"
+                  class="apos-wizard__translation-disclaimer"
+                >
+                  {{ automaticTranslationDisclaimer }}
+                </p>
                 <AposCheckbox
                   v-model="wizard.values.translateContent.data"
                   :field="{ name: 'translate' }"
                   :choice="{
                     value: wizard.values.translateContent.data,
-                    label: $t('apostrophe:automaticTranslationCheckbox')
+                    label: $t('apostrophe:automaticTranslationCheckbox'),
+                    htmlHelp: $t('apostrophe:automaticTranslationCheckboxHelp')
```

Comment:
Are we sure that i18n is not encoding HTML?

Reply:
We do it already and it seems to work, for example [i18n/en.json#54](https://github.com/apostrophecms/apostrophe/blob/1b84327ab0d4952ea800321da0076da2b045e946/modules/%40apostrophecms/i18n/i18n/en.json#L54)